import mysql.connector
from mysql.connector import Error

import ast
with open('dict-list.txt', 'r') as file:
    list = ast.literal_eval(f'[{file.read()}]')
ios1 = list[0]
ios2 = list[1]

a='as'
b='as'
c='as'
d='as'

try:
    connection = mysql.connector.connect(host='localhost',
                                         database='ssssss',
                                         user='xx',
                                         password='xxxx')

   cursor = connection.cursor()
   mySql_insert_query = """INSERT INTO data (intf, ipaddr, status, proto) 
                                VALUES (%s, %s, %s, %s) """

        record = (a,b,c,d)
        cursor.execute(mySql_insert_query, record)

    connection.commit()
    print(cursor.rowcount, "kayıt başarılı bir şekilde gerçekleşti")
    cursor.close()

except mysql.connector.Error as error:
    print("Failed to insert record into Laptop table {}".format(error))

finally:
    if connection.is_connected():
        connection.close()
        print("MySQL bağlantısı sonlandırıldı")
